.github/workflows/basic-continuous-integration.yml

```diff
@@ -31,9 +36,7 @@ jobs:
         run: vendor/bin/phpunit --coverage-clover=coverage.xml
 
       - name: Monitor coverage
```

Comment:
This doesn't appear to have run either, which doesn't quite make sense since this step is being run in a `pull_request`.

Reply:
If you look at the [default values](https://github.com/slavcodev/coverage-monitor-action?tab=readme-ov-file#inputs), I think we can remove a lot of these statements (including `check: true`).

Reply:
Oh, I think it's actually because this GitHub Action needs to change from `on: [ push ]` to `on: [ pull_request ]`.